examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def dj_algorithm(case, n_qubits):
</pre><p>This is a long function worth splitting into multiple smaller ones; as well, it looks like it contains duplicate or similar code from the previous cell that can be extracted out into common functions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Cut down this function!